### PR TITLE
Core/Spells: add slowing effect to Mage spell Frostbolt

### LINKED ISSUE #18932: [master] Mage - Frostbolt issue
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

**Description:** Frostbolt isn't slowing movement speed by 50% for 15 sec.

**Current behaviour:** (Tell us what happens.) Frostbolt isn't slowing movement speed.

**Expected behaviour:** (Tell us what should happen instead.) Frostbolt should slowing movement speed.

**Steps to reproduce the problem:**

1. Select Mage
2. Cast the Frostbolt spell

**Branch(es):** master

**TC rev. hash/commit:** https://github.com/TrinityCore/TrinityCore/commit/1c00f0b2d405d5f9eee4009ca5810e9721cd36ce

**TDB version:**  703.00 + updates

**Operating system:**  Windows 10 64-bit

[//]: # (This template is for problem reports. For other types of report, edit it accordingly.)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)
[//]: # (For fixes containing C++, create a Pull Request.)

### PR DESCRIPTION
**Pull request by HannibalRoG**
[//]: # (***************************)
[//]: # (** FILL IN THIS TEMPLATE **)
[//]: # (***************************)

**Changes proposed:**

-  Add slowing effect to frostbolt of Mage, taken from @Keader  https://gist.github.com/Keader/02eaf2fd3fa32e7c29f808802a52c111

**Target branch(es):** 3.3.5/master

- [ ] 3.3.5
- [x] master

**Issues addressed:** Closes #  (insert issue tracker number)
Closes #18932

**Tests performed:** (Does it build, tested in-game, etc.)
Compiled and tested in game

**Known issues and TODO list:** (add/remove lines as needed)

### CODE REVIEW

**Comment by Edreisantafe** [comment]
From my starter edition account
wowscrnshot_021017_013614

Here it look like Blizzard.

**Comment by streetrat** [comment]
should be valid.
from, tooltip:
Launches a bolt of frost at the enemy, causing $228597s1 Frost damage and slowing movement speed by $205708s1% for $205708d.

File: sql/updates/world/master/RoG_2017_02_09_01_world.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-- Frostbolt
+DELETE FROM `spell_script_names` WHERE `ScriptName` = 'spell_mage_frostbolt';
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES (228597, 'spell_mage_frostbolt');
```
Comment (Keader) [inline sql/updates/world/master/RoG_2017_02_09_01_world.sql]: Missing new line

File: src/server/scripts/Spells/spell_mage.cpp [hunk 2]
```diff
@@ -463,40 +464,39 @@ class spell_mage_focus_magic : public SpellScriptLoader
...
 class spell_mage_frostbolt : public SpellScriptLoader
 {
-   public:
-       spell_mage_frostbolt() : SpellScriptLoader("spell_mage_frostbolt") { }
-
-       class spell_mage_frostbolt_SpellScript : public SpellScript
-       {
-           PrepareSpellScript(spell_mage_frostbolt_SpellScript);
-
-           void RecalculateDamage(SpellEffIndex /*effIndex*/)
```
Comment (Keader) [inline src/server/scripts/Spells/spell_mage.cpp]: it's no needed anymore?

File: src/server/scripts/Spells/spell_mage.cpp [hunk 3]
```diff
@@ -463,40 +464,39 @@ class spell_mage_focus_magic : public SpellScriptLoader
...
-           }
-       };
-
-       SpellScript* GetSpellScript() const override
-       {
-           return new spell_mage_frostbolt_SpellScript();
-       }
+public:
+    spell_mage_frostbolt() : SpellScriptLoader("spell_mage_frostbolt") { }
+
```
Comment (Keader) [inline src/server/scripts/Spells/spell_mage.cpp]: white space xd

File: src/server/scripts/Spells/spell_mage.cpp (hunk 2 above)
Comment (Shauren) [inline src/server/scripts/Spells/spell_mage.cpp]: nope, shatter now only increases critical chance

**Comment by HannibalRoG** [comment]
Any comments?

**Pull request merged**